tests/test_model.py

```diff
@@ -273,6 +273,10 @@ def test_session_key_attribute_without_initialization_vector():
     assert None == obj.session_keys[0].iv
 
 
+def test_parse_custom_media_header():
+    obj = m3u8.M3U8(playlists.PLAYLIST_WITH_CUSTOM_MEDIA_HEADER)
```

Comment:
I think we should expect this test to assert something. I get it does not break, but what happens with the EXT-X-MEDIA-READY? Does it go away? Does it get parsed?